### PR TITLE
Added wasm to the docker image

### PR DESCRIPTION
**Pull request by AndreiNavoichyk**

### CODE REVIEW

File: .github/workflows/build-and-push-docker-image.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - dev-cere
+      - feature/support-wasm-file-copy
```
Comment (AndreiNavoichyk) [inline .github/workflows/build-and-push-docker-image.yml]: Remove

File: scripts/copy-wasm.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export dir_path_to="${1:-./}"
```
Comment (AndreiNavoichyk) [inline scripts/copy-wasm.sh]: rename file to `extract-wasm`.sh

**Review by AndreiNavoichyk — COMMENTED** [review]
1. Add the instructions to README.
2. Add update to the CHANGELOG.
3. Update Dockerfile

File: scripts/copy-wasm.sh (hunk 2 above)
Comment (karthikeyan-cere) [reply scripts/copy-wasm.sh]: renamed

File: .github/workflows/build-and-push-docker-image.yml (hunk 1 above)
Comment (karthikeyan-cere) [reply .github/workflows/build-and-push-docker-image.yml]: Removed

File: scripts/extract-wasm.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export dir_path_to="${1:-./}"
+export image_tag="${2:-latest}"
+docker pull 338287888375.dkr.ecr.us-west-2.amazonaws.com/pos-network-node:"$image_tag"
```
Comment (AndreiNavoichyk) [inline scripts/extract-wasm.sh]: Let's use dockerhub here

File: Dockerfile [hunk 4]
```diff
@@ -19,6 +19,7 @@ LABEL maintainer="team@cere.network"
 LABEL description="This is the optimization to create a small image."
 ARG PROFILE=release
 COPY --from=builder /cerenetwork/target/$PROFILE/cere /usr/local/bin
+COPY --from=builder /cerenetwork/target/release/wbuild/node-runtime /home/cere/wasm-file
```
Comment (AndreiNavoichyk) [inline Dockerfile]: Rename `wasm-file` to `node-runtime-artifacts`

File: scripts/extract-wasm.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export dir_path_to="${1:-./}"
```
Comment (AndreiNavoichyk) [inline scripts/extract-wasm.sh]: Let's use as default `node-runtime-artifacts`

File: scripts/extract-wasm.sh [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+export image_tag="${2:-latest}"
+docker pull 338287888375.dkr.ecr.us-west-2.amazonaws.com/pos-network-node:"$image_tag"
+id=$(docker create "338287888375.dkr.ecr.us-west-2.amazonaws.com/pos-network-node:$image_tag")
+docker cp "${id}":/home/cere/wasm-file "$dir_path_to"
```
Comment (AndreiNavoichyk) [inline scripts/extract-wasm.sh]: Update path here as well

File: scripts/extract-wasm.sh (hunk 3 above)
Comment (karthikeyan-cere) [reply scripts/extract-wasm.sh]: Updated to use dockerhub

File: Dockerfile (hunk 4 above)
Comment (karthikeyan-cere) [reply Dockerfile]: Updated

File: scripts/extract-wasm.sh (hunk 5 above)
Comment (karthikeyan-cere) [reply scripts/extract-wasm.sh]: updated

File: scripts/extract-wasm.sh (hunk 6 above)
Comment (karthikeyan-cere) [reply scripts/extract-wasm.sh]: Updated

File: scripts/extract-wasm.sh [hunk 7]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export dir_path_to="${1:-./node-runtime-artifacts}"
+export image_tag="${2:-latest}"
+docker pull docker pull cerebellumnetwork/pos-node:"$image_tag"
```
Comment (AndreiNavoichyk) [inline scripts/extract-wasm.sh]: remove extra `docker pull`

Comment (karthikeyan-cere) [reply scripts/extract-wasm.sh]: Updated

**Pull request merged**